package.json

```diff
@@ -54,6 +55,10 @@
         "command":"sortLines.sortLinesLineLengthReverse",
         "title": "Sort lines (line length descending)"
       },
+      {
+        "command": "sortLines.sortLinesNatural",
+        "title": "Sort lines (natural)"
```

Comment:
Natural feels a bit ambiguous to me, maybe just numeric would be better? This also needs a dot point in the README

Reply:
> Natural feels a bit ambiguous to me

I get your point. "numerical" was my initial choice, but when I looked into the referenced issue #10, I realized that fixing #10 would also fix #8 as it is more general. The linked wikipedia page also lists various references where this algorithm is referred to as "natural sort" (or natsort). 

> This also needs a dot point in the README

You're right, I totally forgot about that. Will update it later.

Reply:
:+1: Natural sounds good, maybe link to wikipedia in the README description to clear up ambiguities :smiley: